### PR TITLE
Add window functions to coverage map

### PR DESCRIPTION
**Pull request by pramodsatya**
Cherrypicks changes from #4158 that were subsequently reverted.

### CODE REVIEW

**Comment by pramodsatya** [comment]
@aditi-pandit, it was reverted in #3927 and from the PR description it appears to be an inadvertent error.

File: velox/functions/CoverageUtil.cpp [hunk 1]
```diff
@@ -162,7 +167,7 @@ void printTableCss(
   out << "    table.coverage th {background-color: lightblue; text-align: center;}"
       << std::endl;
   out << "    table.coverage "
-      << "td:nth-child(" << numScalarColumns + 1 << ") "
```
Comment (zacw7) [inline velox/functions/CoverageUtil.cpp:165]: This line needs to be kept.

**Review by zacw7 — APPROVED** [review]
LGTM. Only one minor change.

**Review by aditi-pandit — APPROVED** [review]
Thanks Pramod.

**Comment by aditi-pandit** [comment]
@zacw7 : Please can you help merge this PR. It has been pending for long time. 

Many new functions have been included since the last update also.

**Comment by zacw7** [comment]
> @zacw7 : Please can you help merge this PR. It has been pending for long time.
> 
> Many new functions have been included since the last update also.

Sorry for delay. Didn't realize it's waiting for me to perform the merge. There's some ongoing issue preventing the PR being synced. Will have it merged once the issue is fixed.

**Pull request closed without merging**